docs/docker.md

```diff
@@ -2,15 +2,33 @@
 
 ## Overview
 
-Docker can automate much of the configuration for you.
+Docker packages software into virtual containers so it can run on any Linux, Windows, or macOS computer. It can automate much of the configuration for you.
```

Comment:
We would be talking about applications for our use case. We don't need the virtual in front of containers, "containers" should be fine.

Looking at the second statement now we could probably remove "can." I think something starting with "It automates much ..." would be much clearer.